part-time-schedule/week-5.md

```diff
@@ -0,0 +1,62 @@
+---
+description: Mastering Lightning and Node Management Workshop
+---
+
+# Week 5
+
+This week, we'll continue with "Mastering Lightning" book, holding Discussions in small groups as you answer discussion questions. We'll also be having workshops on Node Management.
+
+Also, we'll be working on more projects or expanding, refining or improving a project you have already started. 
+
+As usual, a web-reader-friendly hosted version of Mastering Lightning Book can be found at [lnbook.256k1.dev](https://lnbook.256k1.dev), which can be accessed for educational purposes using username and password `qala:lightning`.
+
+## Prerequisites
+
+* All the previous weeks' developer environments for both Bitcoin and Lightning Network.
+* All the previous chapters of Mastering Lightning Book.
+
+## Goals
+
+At the end of the week, we hope that you will:
+
+* Gain practical knowledge of how to manage a Mining Node
```

Comment:
"Mining Node" is wrong, I think? Not sure if the workshop is about Bitcoin or Lightning or both?